README.md

```diff
@@ -15,8 +15,11 @@ is a very simple task, but you have to wait quite some time.*
 Once having it on your machine, you can run it with:
 ```bash
 $ mkdir ./result
-$ docker run -it --volume="$PWD/result:/result" mikelangelo/capstan-packages
+$ docker run -it --priviledged --volume="$PWD/result:/result" mikelangelo/capstan-packages
 ```
+*NOTE: the `--priviledged` flag is needed in order to enable KVM inside container. Container will
+still work without the flag, but recipe tests will last longer.*
```

Comment:
No, why? Lines 20 and 21 are meant to be italics so they are wrapped between two single `*`.